src/test/java/lotto/domain/LottoNumbersTest.java

```diff
@@ -0,0 +1,36 @@
+package lotto.domain;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.List;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.*;
+public class LottoNumbersTest {
+    @Test
+    public void 생성() {
+        assertThat(new LottoNumbers(List.of(1, 2, 3, 4, 5, 6))).isEqualTo(new LottoNumbers(List.of(1, 2, 3, 4, 5, 6)));
```

Comment:
```suggestion
        assertThat(new LottoNumbers(1, 2, 3, 4, 5, 6)).isEqualTo(new LottoNumbers(1, 2, 3, 4, 5, 6));
```
가변 인자를 가지는 생성자도 추가해 위와 같이 구현 가능하도록 지원하면 어떨까?